src/components/Profile/ProfileCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { ProfileData } from '../../types/profileColumnData';
+
+interface ProfileCardProps {
+  userData: ProfileData;
+  onEditClick: () => void;
+}
+
+const ProfileCard: React.FC<ProfileCardProps> = ({ userData, onEditClick }) => {
+  return (
+    <div className="flex flex-col items-center w-full sm:w-96 md:w-80 lg:w-96 mx-auto">
+      {/* Profile Picture */}
```

Comment:
```suggestion
      {/* Profile Picture */}
```
Similarly, this one and all comments in this file can be removed.